src/main/java/uk/gov/digital/ho/hocs/casework/api/CorrespondentService.java

```diff
@@ -49,6 +64,15 @@ Correspondent getCorrespondent(UUID caseUUID, UUID correspondentUUID) {
         }
     }
 
+    Set<CorrespondentTypeDto> getCorrespondentTypes(UUID caseUUID){
+        log.debug("Getting all Correspondent Types for Case: {}", caseUUID);
+        CaseData caseData = caseDataRepository.findByUuid(caseUUID);
```

Comment:
We only need 'caseType' here. Let's create a method that retrieves case type for case uuid.

Reply:
Do you mean add a method to the CaseData Repository that returns only the Type string, rather than the whole CaseData object?

Reply:
Yes, similar to this one 'String getCaseRef(UUID uuid);'.